### DISCUSSION TITLE
Looking for paid support agreements

### DISCUSSION THREAD

**Discussion by PG-MBR** [QuestPDF/QuestPDF#881] [General]
Hello dear QuestPDF-Community,

I am working as part of a small development team (<10 people) and we are looking to start to use QuestPDF. While we appreciate the ability to ask questions and get support for free here on Github, we are also looking to be able to get paid support in case something urgent comes up that e.g. impacts our production environments.

I understand after reading the discussion in https://github.com/QuestPDF/QuestPDF/discussions/491 that QuestPDF itself is not going to offer paid support, but maybe there are interested third party agencies reading here willing to consider such a support agreement?

Cheers
Michael Breuer

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for your interest in QuestPDF.

At this time, QuestPDF does not offer paid support agreements. For any questions or issues, the GitHub Discussions and Issues remain the best channels for community and maintainer support.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
